fedmsg.d/fmn.py

```diff
@@ -19,11 +19,19 @@
     "fmn.rules.utils.use_pkgdb2": False,
     "fmn.rules.utils.pkgdb2_api_url": "http://209.132.184.188/api/",
     "fmn.rules.cache": {
```

Comment:
Ill try this shortly without redis and see if it will still work since i fixed the old default in an earlier pr